third_party.bzl

```diff
@@ -23,6 +23,10 @@ _deps = [
     _dependency("org.eclipse.xtext:org.eclipse.xtext.xbase.lib:2.19.0"),
     _dependency("commons-cli:commons-cli:jar:1.4"),
     _dependency("org.jetbrains:annotations:jar:20.1.0"),
+    _dependency("io.vavr:vavr:0.9.0"),
+    _dependency("org.projectlombok:lombok:1.18.4"),
+    _dependency("org.apache.logging.log4j:log4j-api:2.6.1"),
+    _dependency("org.apache.logging.log4j:log4j-core:2.6.1"),
```

Comment:
Why did you add Lombok and a separate logging tool and not Lombok's Slf4j? Is Lombok used for something else?

Reply:
I forgot to delete this dep after my experiments